tests/stdlib/tmath.nim

```diff
@@ -176,13 +176,6 @@ block:
       doAssert round(-54.352) == -54.0
       doAssert round(0.0) == 0.0
 
-    block: # splitDecimal() tests
-      doAssert splitDecimal(54.674).intpart == 54.0
-      doAssert splitDecimal(54.674).floatpart ==~ 0.674
-      doAssert splitDecimal(-693.4356).intpart == -693.0
-      doAssert splitDecimal(-693.4356).floatpart ==~ -0.4356
-      doAssert splitDecimal(0.0).intpart == 0.0
-      doAssert splitDecimal(0.0).floatpart == 0.0
```

Comment:
Please don't remove these (I assume it's an artifact from rebasing?)

Reply:
yeah